source/MaterialXRender/CMakeLists.txt

```diff
@@ -30,35 +26,44 @@ if(MATERIALX_BUILD_SHARED_LIBS AND MSVC)
     target_sources(${MATERIALX_MODULE_NAME} PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/version.rc)
 endif()
 
+set_target_properties(
+    ${MATERIALX_MODULE_NAME} PROPERTIES
+    OUTPUT_NAME ${MATERIALX_MODULE_NAME}${MATERIALX_LIBNAME_SUFFIX}
+    COMPILE_FLAGS "${EXTERNAL_COMPILE_FLAGS}"
+    LINK_FLAGS "${EXTERNAL_LINK_FLAGS}"
+    INSTALL_RPATH "${MATERIALX_SAME_DIR_RPATH}"
+    VERSION "${MATERIALX_LIBRARY_VERSION}"
+    SOVERSION "${MATERIALX_MAJOR_VERSION}")
+
 target_link_libraries(
     ${MATERIALX_MODULE_NAME}
     MaterialXGenShader
     ${CMAKE_DL_LIBS})
 
 if(MATERIALX_BUILD_OIIO)
     set(OPENIMAGEIO_ROOT_DIR ${MATERIALX_OIIO_DIR})
-    list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/External/OpenImageIO")
-    find_package(OpenImageIO REQUIRED)
-    if(OPENIMAGEIO_FOUND)
-        include_directories(${OPENIMAGEIO_INCLUDE_DIR})
-        target_link_libraries(${MATERIALX_MODULE_NAME} ${OPENIMAGEIO_LIBRARIES})
+    find_package(OpenImageIO CONFIG REQUIRED)
+    if(OpenImageIO_FOUND)
+        message("Found OpenImageIO")
+        include_directories(${OpenImageIO_INCLUDE_DIR})
+        target_link_libraries(${MATERIALX_MODULE_NAME} OpenImageIO::OpenImageIO OpenImageIO::OpenImageIO_Util)
```

Comment:
It's customary here to put either PUBLIC or PRIVATE, depending on whether the OpenImageIO types or functions are exposed through your APIs (I expect not, so should probably be PRIVATE).